### PR TITLE
remove unused proc and mention of interactive session

### PR DESCRIPTION
**Pull request by lucaferranti**
Fixes https://github.com/chapel-lang/chapel/issues/25071

### CODE REVIEW

File: doc/rst/tools/mason/guide/basicusage.rst [hunk 1]
```diff
@@ -14,8 +14,7 @@ To initialize a new mason package, run ``mason new``. The same can also be done
     cd newPackage
     mason init
 
-
-This starts an interactive session which walks a user through the process of creating a project using Mason. This is highly recommended for new users.
+This setups the folder to have the structure of a mason package. Particularly, it creates a `Mason.toml` file and the `src` folder with the package main module.
```
Comment (bmcdonald3) [inline doc/rst/tools/mason/guide/basicusage.rst:17]: "setups the", I think you mean "sets up a"? So, "This sets up a folder to have..."

**Review by bmcdonald3 — APPROVED** [review]
Awesome, thank you for putting this together Luca, much appreciated! Just one minor comment about some wording.

File: doc/rst/tools/mason/guide/basicusage.rst (hunk 1 above)
Comment (lucaferranti) [reply doc/rst/tools/mason/guide/basicusage.rst]: yes! thanks fixed

**Comment by bmcdonald3** [comment]
Thanks again for putting this together Luca!

**Pull request merged**